Makefile

```diff
@@ -1,6 +1,10 @@
 export XDG_CONFIG_HOME ?= $(HOME)/.config
 
-FISHTAPE = $(XDG_CONFIG_HOME)/fish/functions/fishtape.fish
+# In case someone wants to run the tests and already has Fishtape,
+# don't install it again, use the already installed one.
```

Comment:
This is because I installed Fishtape via Fundle, so I kept having to remove `~/.config/fish/functions/fishtape.fish` ever time I ran `make test`.

Reply:
You need to Fish 3.0 to run Fishtape.

Reply:
I suspected there was something of the sort.
But then doesn't that mean that we should also change the runner's settings ?
Also the comment box doesn't show this well but the line I changed here uses Fish `functions` to check the presence of Fishtape and find its path if one has it; or uses `$HOME/.config/fish/functions` as usual..

Reply:
There's no reason to not use at least Fish 3.0 IMO. [Here's how Fishtape tests itself with itself](https://github.com/jorgebucaran/fishtape/blob/e6e5fc23dd062ee5ed11828951bf2c85d6798db5/.github/workflows/ci.yml#L19-L21).